extensions/cms/src/apiWrapper.ts

```diff
@@ -269,20 +282,17 @@ export class ApiWrapper {
 				authTypeChanged: true
 			}
 		};
-		return this.openConnectionDialog(['MSSQL-CMS'], initialProfile, undefined).then((connection) => {
+		return this.openConnectionDialog([cmsProvider], initialProfile, { saveConnection: false }).then(async (connection) => {
```

Comment:
>saveConnection [](start = 68, length = 14)

As discussed please add comments for each of these kind of things so we can track why things are being set in a certain way (especially with a confusing name like saveConnection)